storage_service/templates/snippets/packages_table.html

```diff
@@ -9,6 +9,7 @@
         <th class="sortable">{% trans "Type" %}</th>
         <th class="sortable">{% trans "Replica Of" %}</th>
         <th class="sortable">{% trans "Status" %}</th>
+        <th class="sortable">{% trans "Stored" %}</th>
```

Comment:
This makes the DataTable column sortable in the frontend, but the backend class that handles the sorting needs to be updated too. The implementation is a bit clunky and not the most efficient (cough since I wrote it cough...), but the change should be:

1. Add a new column index to the `SORT_KEY_HELPERS_MAPPING` dictionary which value should be the name of a method that receives a `Package` instance and returns a sorting key for it. The new column index needs to be inserted before the fixity ones.
2. Add that named method to the class. As you can see in the `sort_by_fixity_date_key` method for example, for datetimes, since the attribute is nullable we need to consider that it's not possible to compare datetimes and `None`. In that particular case I decided to use `datetime.now`.

I've created https://github.com/artefactual/archivematica-storage-service/commit/4e5c7f42eea40543807dddfde090380ec0af3966 as an example of how that might look. I hope it's useful.